doc/usage/index.rst

```diff
@@ -25,3 +25,60 @@ Once the daemon is running, you need to choose which front-end's CLI to use:
 - :doc:`bfcli`: ``bpfilter``-specific CLI, developed as part of the project. ``bfcli`` supports new ``bpfilter`` features before other CLIs as it's used for development. It allows for a more flexible rule definition: you can use a combination of filters and hooks that might not be possible with other CLIs. However, it doesn't support ``nftables`` or ``iptables`` rules format.
 - :doc:`nftables`: requires a custom version of the ``nft`` binary with ``bpfilter`` support (see below), and support for new ``bpfilter`` features is usually a bit delayed.
 - :doc:`iptables`: similar to ``nftables``, however ``iptables`` has been deprecated globally in favor of ``nftables``.
+
+Example Usage
+-------------
+
+.. note::
+
+	This is only to be used as an example and a more interactive test to familiarize you with ``bpfilter``, more in-depth information can be found throughout the docs.
+
+Initialize ``bpfilter`` daemon
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: bash
+
+	> cd path/to/bpfilter/
+	> sudo ./build/output/bpfilter -t -v debug --no-iptables
```

Comment:
Use long options here, it's easier to understand for people learning about `bpfilter`.